index.js

```diff
@@ -96,6 +107,50 @@ function parse(str, options) {
  */
 
 function serialize(name, val, options) {
+  if( typeof name === 'object') {
+    var cookies = name;
+    var serializeOptions = val;
+
+    var cookieNames = Object.keys(cookies);
+    if(0 === cookieNames.length) {
+      return '';
+    } else if(cookieNames.length > 1) {
+      // If there are more than one cookies to serialize, only allow
+      // an encoding option to be set
+      var opt = serializeOptions || {};
+      serializeOptions = {
```

Comment:
This doesn't really make any sense. People supplying the object as a variable would have no way to understand if their options are going to be honored. Besides, it seems perfectly valid to want to set all the cookies to have `httpOnly: true`, `secure: true`, etc.